### ISSUE TITLE
Improve documentation about how to configure Twilio for Chatwoot

### ISSUE THREAD

**Issue by Vadorequest** [chatwoot/chatwoot#4547]
**Is your enhancement request related to a problem? Please describe.**
I got lost in the process of configuring Twilio for Chatwoot. 
I successfully created the Twilio Inbox on Chatwoot, but then I was asked to configure Twilio callbacks and I don't know where to do that. 

Video: https://youtu.be/4UpwUL4pUEg

**Describe the solution you'd like**
Once the Inbox has been added, a video tutorial about where to go in Twilio to configure the callbacks would be much helpful.

**Comment by pranavrajs** [comment]
@Vadorequest Thanks for the report, this UX needs to be improved. Can you please if this document is helpful? https://www.chatwoot.com/docs/product/channels/whatsapp/twilio

**Comment by Vadorequest** [comment]
Yes, this documentation helped me. But I had to ask the support which send me the same link, it'd have been better to rather have that link in the last step of the process.

**Comment by sojan-official** [comment]
@Vadorequest  thanks for the feedback, we will look further into how we can improve the product documentation for this.

**Comment by stonecharioteer** [comment]
This issue is being closed due to lack of activity for over a year. [For more details, visit this discussion.](https://github.com/orgs/chatwoot/discussions/12810)
